src/components/Form/Location/Location.jsx

```diff
@@ -607,6 +608,11 @@ export default class Location extends ValidationElement {
             required={this.props.required}
           />
         )
+      // XXX This is used in ContactItem, which is foreign business contacts,
+      // would we ever need domestic fields for this?
```

Comment:
Yeah, domestic addresses are relevant here because the form asks you where (for instance) the foreign job offer was made – and even though the job might be foreign, the offer might have been made at a location in the US.

Reply:
Ah yes, I see that on the matrix. Thanks!